doc/reverse_engineering/networking/08-movement.md

```diff
@@ -122,7 +122,7 @@ Consecutive movement command.
 >`00 78 27 43` &mdash; x_coord<br/>
 >`55 cd 39 43` &mdash; y_coord
 
-## Setting Waypoints
+## Setting waypoints
```

Comment:
This is unnecessary because titles should be capitalized.

Reply:
kk, made the style consistent also in the other documents